projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -543,13 +547,16 @@ public Response getStandings(
 		try {
 			logger.info("Standings requested by team " + userInformation.getMyClient().getLoginName());
 			
+			xlog(logger, "debug 22 Start" );
```

Comment:
Is the intent to leave this debugging log stuff in production code? If not, it should be removed.